### ISSUE TITLE
Constant grep error when command completion

### ISSUE THREAD

**Issue by MircoBlitz** [Bash-it/bash-it#1210]
Hi, I get an annoying and constant grep error when using bash-it on a Mac OsX (Sierra, Highe Sierra and Mojave). 

ssh lind<tab> becomes.
`ssh lindusage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]
worm
`

There are used non standard flags in grep :-(

Best regards
Mirco

**Comment by LanikSJ** [comment]
I'm not certain but part of me thinks we should be looking at https://github.com/Bash-it/bash-it/pull/1205

**Comment by nwinkler** [comment]
Yes, it's the `grep -P` on line 16 that seems to be causing this:

https://github.com/Bash-it/bash-it/blob/f121bec2a1f9d32829f8d362644504a1392769fb/completion/available/ssh.completion.bash#L16

The `-P` flag seems to enabled Perl regex syntax - anybody know whether there is a similar option for macOS/BSD-style `grep`, or a way to rewrite the command without using `-P`?

Copying @con-f-use since he provided the original PR (#1205) that introduced this issue. Can you please outline why the `-P` is required in this case?

**Comment by con-f-use** [comment]
Ah yes, it should work  without `-P` on most systems. I had a more complicated expression there before, where the flag was needed. Sorry for that. didn't know there was any version of grep that is missing `-P`. 

I don't use MacOS, so no idea how grep behaves there. All the expression is meant to do is find lines that start with any number of whitespaces and the word "Include "

**Comment by nwinkler** [comment]
Thanks for the response! Would you mind providing a PR that does this without `-P`? Really appreciate your help with this!

**Comment by nwinkler** [comment]
I gave that a try on macOS, and it seems to work:
* Comes back empty if the `config` file does not include any lines starting with `Include`.
* Shows the lines if any were found.
